doc/ref_arch/openstack/chapters/chapter05.md

```diff
@@ -76,7 +93,7 @@ This chapter presents a consolidated set of OpenStack Service APIs corresponding
 ## 5.3. Consolidated Set of APIs.
 
 ### 5.3.1. OpenStack Interfaces
-This section illustrates some of the Interfaces provided by OpenStack; the exhaustive list of APIs is available at https://docs.openstack.org/api-ref/ and CNTT standard is to use the latest version v3. OpenStack REST APIs are simple to interact with using either of two options. Clients can either call the APIs directly using the HTTP or REST library, or they can use one of the many programming language specific cloud libraries. 
+This section illustrates some of the Interfaces provided by OpenStack; the exhaustive list of APIs is available at https://docs.openstack.org/api-ref/ and CNTT standard is to use the latest version v3. OpenStack REST APIs are simple to interact with using either of two options. Clients can either call the APIs directly using the HTTP or REST library, or they can use one of the many programming language specific cloud libraries.
```

Comment:
The latest version `v3` is incorrect to say as some of the services has no `v3` available.

Reply:
@deebhatia You're right. 
I will open a new issue to solve this point as it isn't link to the current PR. 
And if you agree, it will be up to you to propose a PR to modify the text.

Reply:
Sure, happy to do. Please assign it to me.

Reply:
@deebhatia Done! Issue #561 created